fragments/platform/rhel10/payload/ostreecontainer.ks

```diff
@@ -0,0 +1 @@
+ostreecontainer --no-signature-verification --transport=registry --url=quay.io/centos-bootc/centos-bootc:stream10
```

Comment:
This is not available and probably won't be anytime soon, so tests depending on this fragment will fail.

Reply:
That's ok, we will see it and address it separately.